lib/Doctrine/DBAL/Driver/PDOConnection.php

```diff
@@ -30,6 +30,11 @@
 class PDOConnection extends PDO implements Connection, ServerInfoAwareConnection
 {
     /**
+     * @var string
+     */
+    private $lastInsertId = '0';
```

Comment:
Because I need to know how to retrieve the last insert ID in the statement object? Simply passing the LastInsertId container won't help in `PDOStatement::execute()`